src/bin/inx-chronicle/process.rs

```diff
@@ -1,28 +1,19 @@
 // Copyright 2022 IOTA Stiftung
 // SPDX-License-Identifier: Apache-2.0
 
-pub async fn interrupt_or_terminate() {
+pub async fn interrupt_or_terminate() -> eyre::Result<()> {
     #[cfg(unix)]
     {
         use tokio::signal::unix::{signal, SignalKind};
-
-        let mut sigterm = signal(SignalKind::terminate()).expect("cannot listen to `SIGTERM`");
-        let mut sigint = signal(SignalKind::interrupt()).expect("cannot listen to `SIGINT`");
-
+        let mut terminate = signal(SignalKind::terminate())?;
```

Comment:
Rather than remove these strings, can we replace them with `eyre` wrappers?

```
let mut terminate = signal(SignalKind::terminate()).map_err(|e| eyre!("cannot listen to `SIGTERM`: {e}"));
```